react/features/chat/components/web/ChatMessage.js

```diff
@@ -16,6 +16,20 @@ import PrivateMessageButton from '../PrivateMessageButton';
  * Renders a single chat message.
  */
 class ChatMessage extends AbstractChatMessage<Props> {
+
+    /**
+     * Checks if the given string is a valid URL.
+     *
+     * @param {string} input - The inputstring given.
+     * @returns {boolean} True when the input is a valid string.
+     */
+    isValidUrl(input) {
+        const res = input.match(
+            /(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_+.~#?&/=]*)/g);
```

Comment:
Not a fan of detecting links like this. I think this could be accomplished with a decorator on the linkify library. Can you please take a look at that?

Reply:
We've changed it around to use Linkify in the renderers prop in the React-Markdown component. This way Linkify will run on plaintext.